extract/db_download_files.py

```diff
@@ -0,0 +1,27 @@
+import dropbox
+from lib.get_dropbox import get_files, get_folders
+from lib.path import data_file_path
+
+
+with open(data_file_path("drop_box/token.txt", "r")) as f:
+    access_token = f.read()
+
+
+dbx = dropbox.Dropbox(access_token)
+
+
+folders = get_folders(
+    dbx, "/IPNO/data/Orleans/Data Collected/New Orleans Civil Service Commission/"
+)
+
+# print folder IDs
+print(folders)
+
+# # folder_id for transcripts folder within the directory above
+folder_id = "id:8ceKnrnmgi0AAAAAAAA5bA"
+
+# download directory
+download_dir = data_file_path("raw/new_orleans_pd/appeal_hearing_pdfs/")
+
+# # print('Obtaining list of files in target directory...')
+get_files(dbx, folder_id, download_dir)
```

Comment:
Best practice is to put every line from line 6 and below (basically everything excepts imports) in the main block:
```python
if __name__ == '__main__':
    ...
```
That way these statements are only executed when you call the script directly, not when you merely import them. But even if you never import a script you should still use the main block as it communicates your intention clearer.

Reply:
Got it.